src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -809,15 +810,53 @@ def test_containerapp_managed_service_binding_e2e(self, resource_group):
         self.cmd('containerapp create -g {} -n {} --environment {} --bind {}:{},database={},username={},password={}'.format(
             resource_group, ca_name, env_name, mysqlserver, flex_binding, mysqldb , mysqlusername, mysqlpassword))
         self.cmd('containerapp show -g {} -n {}'.format(resource_group, ca_name), checks=[
+            JMESPathCheck("properties.provisioningState", "Succeeded"),
             JMESPathCheck('length(properties.template.containers[0].env[?name==`AZURE_MYSQL_HOST`])', 1)
         ])
 
         self.cmd('containerapp update -g {} -n {} --bind {},database={},username={},password={}'.format(
             resource_group, ca_name, postgresqlserver, postgresqldb , postgresqlusername, postgresqlpassword))
         self.cmd('containerapp show -g {} -n {}'.format(resource_group, ca_name), checks=[
+            JMESPathCheck("properties.provisioningState", "Succeeded"),
             JMESPathCheck('length(properties.template.containers[0].env[?name==`AZURE_MYSQL_HOST`])', 1),
             JMESPathCheck('length(properties.template.containers[0].env[?name==`AZURE_POSTGRESQL_HOST`])', 1)
         ])
+        self.cmd('containerapp update -g {} -n {} --bind kafkaconfluent,bootstrap_server={},kafka_key={},kafka_secret={}'.format(
```

Comment:
Please rerun test_containerapp_dev_service_binding_e2e , and I recommend to add test for execute same command twice,